dockerfiles/alpine/obam/Dockerfile

```diff
@@ -44,51 +47,74 @@ ARG MOTD='printf "\n\
 ENV ENV=${USER_HOME}"/.ashrc"
 
 # create the non-root user and group and set MOTD login message
```

Comment:
@AmilaSamith do we require lines 49-50? As per my observation don't need them, please correct me if I am wrong.

If yes you can remove them.

Please check the other Dockerfiles as well.

Reply:
Yes we need them @chirangaalwis as we need to override MOTD of base image